digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/util/DataFixture.scala

```diff
@@ -1002,7 +1022,50 @@ object DataFixture {
       }
     }
 
-    println("\nFill Road Width in missing and incomplete road links")
+    println("Complete at time: " + DateTime.now())
+  }
+
+  def verifyInaccurateSpeedLimits(): Unit = {
+    println("Start inaccurate SpeedLimit verification\n")
+    println(DateTime.now())
+
+    val polygonTools: PolygonTools = new PolygonTools()
+    val dao = new OracleLinearAssetDao(null, null)
+
+    //Expire all inaccuratedAssets
+    OracleDatabase.withDynTransaction {
+      inaccurateAssetDAO.deleteAllInaccurateAssets(SpeedLimitAsset.typeId)
+    }
```

Comment:
Maybe it's a good idea extend the transaction until the end of all procedure, instead of having two Transactions, one for delete all and other to create the incorrect speed limits on database.
With that, we ensure the delete only it's executed if the batch process finishes without problems, in case some error happens, it's all reverted and we don't delete the records on the database.

Reply:
Maybe if we use the same Transaction for all process we will have some abend. I thing that is a good ideia open and close the Transaction for each municipality